index.d.ts

```diff
@@ -1110,6 +1116,9 @@ declare module powerbi {
         /** The selector that identifies this object. */
         selector: Selector;
 
+        /** Additional selector used for conditional formatting at the static level to optimize perf. */
```

Comment:
'for performance optimization' ?

Reply:
Sure, copy-pasted from here: src/Clients/VisualHostCore/services/serializers/visualObjectInstanceSerializer.ts